doc_src/cmds/path.rst

```diff
@@ -0,0 +1,359 @@
+.. _cmd-path:
+
+path - manipulate and check paths
+=================================
+
+Synopsis
+--------
+
+::
+
+    path base [(-z | --null-in)] [(-Z | --null-out)] [(-q | --quiet)] [PATH...]
+    path dir  [(-z | --null-in)] [(-Z | --null-out)] [(-q | --quiet)] [PATH...]
```

Comment:
For now I decided against it because it's simply not long enough, and there's a bunch of general information - e.g. `-z` applies to *every single* subcommand, so it's mentioned once in the general section.